Code-4-Community/speak-for-the-trees-frontend
# Global style

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/702429655/views/12631042/pulses/829576071)

What benefit does this bring to the end user? Or, what benefit does this bring to developers working in the codebase?

This allows developers to use antd components like text and buttons with the automatic styles that are used throughout the application

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers.

All of the variables are located within the craco.config.js file. The colors are at the top of the page and they correspond to different greens and greys that exist in the figma designs. Any of these values can be overridden in less files, see the info-card.less file for an example of overriding antd global styles.

## Screenshots

Provide screenshots of any new components, styling changes, or pages.

Here is what one of the template page looks like with the new theme. No code or styling was changed on this page, the new colors come from the global style

![theme](https://user-images.githubusercontent.com/19194912/99156721-41bdc000-2691-11eb-950a-2cdebf72a40c.PNG)

## Verification Steps

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.

Checked the styling on all of the current pages, removed instances of p and h from application to ensure styling works everywhere.
